meerkat_nest/config.py

```diff
@@ -31,6 +35,22 @@
 spec.loader.exec_module(country_config_module)
 country_config = country_config_module.country_config
 
+salt_file = os.environ.get("SALT", None)
+salt = None
+
+if salt_file:
+    try:
+        f = open(salt_file, 'r')
+        salt = f.read()
+        salt = salt.encode('utf-8')
+    except FileNotFoundError as e:
+        logging.error(e)
+
+if not salt:
+    logging.warning("No salt file defined, using a random string")
+    salt = hashlib.sha256(str(random.random()).encode('utf-8')).hexdigest()
```

Comment:
I don't think that's a valid error recovery behaviour here. After nest restart records stored in RDS will be broken.
I see two ways:
1) fail here and throw exception
2) generate a random salt and store it in a file within default directory
What do you think?